Makefile.refactor.make

```diff
@@ -48,7 +48,7 @@ clean-glide:
 
 .PHONY: cleanall-glide
 cleanall-glide: clean-glide
-	@rm -rf .glide
+	@rm -rf .glide glide.lock
```

Comment:
the issue with the cleanup of the glide.lock, is that you always end up with a modified workspace: even if the pinned commits are the same, the lock file is updated with a new timestamp.